libraries/arrow-convert/src/from_impls.rs

```diff
@@ -18,6 +18,18 @@ impl From<arrow::array::ArrayRef> for ArrowData {
     }
 }
 
+impl From<ArrowData> for arrow::array::ArrayData {
+    fn from(value: ArrowData) -> Self {
+        value.0.to_data()
+    }
+}
+
+impl From<arrow::array::ArrayData> for ArrowData {
+    fn from(value: arrow::array::ArrayData) -> Self {
+        Self(make_array(value))
+    }
+}
```

Comment:
Could you clarify the use case for _constructing_ an `ArrowData` instance in operator/node code? If I remember correctly, we only use `ArrowData` to provide addiitonal conversion methods for inputs.

Reply:
It was more for testing indeed. I can remove it